lib/auth0/api/v2/jobs.rb

```diff
@@ -76,11 +79,12 @@ def jobs_path
         end
 
         # Map array of field names for export to array of objects
-        # @param fields [array] Field names to be included in the export
-        
-        # @return [array] Returns the fields mapped as array of objects for the export_users endpoint
+        # @param fields [array] Field names to be included in the export
+
+        # @return [array] Returns the fields mapped as array of objects for the export_users endpoint
         def fields_for_export(fields)
-          return nil if fields.nil? || fields.empty?
+          return nil if fields.to_s.empty?
```

Comment:
why the change here?

Reply:
Simplification ... I think it was suggested by the formatting tool `rubocop`